website/versioned_docs/version-1.1/plugins/optuna_sweeper.md

```diff
@@ -100,6 +99,17 @@ You can also override the search space parametrization:
 python example/sphere.py --multirun 'x=interval(-5.0, 5.0)' 'y=interval(0, 10)'
 ```
 
+You might find the `optimization_results.yaml` file (i.e. best params and best value) under `multirun` logs folder:
+
+```yaml
+name: optuna
+best_params:
+  x: 0.0
+  'y': 0
```

Comment:
is this a typo? why does y have single quotes around it?

Reply:
To disambiguate the string 'y'  from the yaml boolean y.